src/main/java/com/cloudbees/jenkins/plugins/bitbucket/endpoints/BitbucketServerEndpoint.java

```diff
@@ -91,11 +91,13 @@
      * @param manageHooks   {@code true} if and only if Jenkins is supposed to auto-manage hooks for this end-point.
      * @param credentialsId The {@link StandardCredentials#getId()} of the credentials to use for
      *                      auto-management of hooks.
-     */
+     * @param bitbucketJenkinsRootUrl The custom (or empty for global setting) Jenkins Root URL
+     *                      auto-management of hooks.
+      */
     @DataBoundConstructor
     public BitbucketServerEndpoint(@CheckForNull String displayName, @NonNull String serverUrl, boolean manageHooks,
```

Comment:
Same as previously I would prefer it was using a `@DataBoundSetter` for bitbucketJenkinsRootUrl and than create another constructor that is not databound to allow receiving bitbucketJenkinsRootUrl on constructor if it makes coded behavior better.